pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -1906,19 +1916,19 @@ func nodeUsingCSIPlugin(og OperationGenerator, spec *volume.Spec, nodeName types
 
 }
 
-func translateSpec(spec *volume.Spec) (*volume.Spec, error) {
+func translateSpec(og OperationGenerator, spec *volume.Spec) (*volume.Spec, error) {
```

Comment:
I tried doing that but unfortunately `OperationExecutor` contains the `OperationGenerator interface` instead of the `operationGenerator struct`. I wasn't too happy about making these exported functions of the `OperationGenerator` basically since the interface is very minimal right now. I'm kind of open to both. Also considered making the inputs more specific like instead of `og` to have `VolumePluginManager` and `CSITranslationLibrary` interfaces passed in since those are the actual things that we need and is more specific so seemed like better code practice.

/cc @misterikkit

Reply:
As above, I think it makes more sense to consume the translator as an arg, since there's a bunch of stuff accessible through `og` that this func doesn't need.